ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -489,9 +489,14 @@ gcloud deployment-manager deployments delete -q "${INFRA_ID}-bootstrap"
 echo "$(date -u --rfc-3339=seconds) - Approving the CSR requests for nodes..."
 function approve_csrs() {
   while true; do
-    oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
-    sleep 15 & wait
-    continue
+    if [[ ! -f /tmp/install-complete ]]; then
```

Comment:
We also need to touch this file when the install completes.

Reply:
Wouldn't this be taken care of by the wait-for install-complete command? Or is it something else?

Reply:
Something else.
https://github.com/openshift/release/blob/eba0c278f39973fec5cd42b97ec7e25ef3104d37/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L2215